src/api/services/soccer.service.ts

```diff
@@ -0,0 +1,45 @@
+import { axiosInstance } from '../config/axios-instance';
+
+export const ROUTE_PREFIX = '/soccer';
+
+export const SoccerService = {
+    async getCountries() {
+        try {
+            const { data } = await axiosInstance.get(`${ROUTE_PREFIX}/countries`);
```

Comment:
```suggestion
 const { data } = await axiosInstance.get<Country[]>(`${ROUTE_PREFIX}/countries`);
```
Add type to generic `.get` function to have typed reponse data and not just type `any`.

Make it on all methods in this `SoccerService` object